addons/project/views/project_views.xml

```diff
@@ -1274,9 +1274,9 @@
                                 </tree>
                             </field>
                         </page>
-                        <page name="recurrence" string="Recurrent"  groups="project.group_project_recurring_tasks">
+                        <page name="recurrence" string="Recurrent" groups="project.group_project_recurring_tasks">
                             <label for="recurring_task" />
-                            <field name="recurring_task" attrs="{'invisible': ['|', ('allow_recurring_tasks', '=', False), ('active', '=', False)]}"/>
+                            <field name="recurring_task" class="ms-5" attrs="{'invisible': ['|', ('allow_recurring_tasks', '=', False), ('active', '=', False)]}"/>
```

Comment:
Did you try to remove the label and put this field in a group?

Reply:
Yes, It can work but we can't do it in stable because It might break customisation :thinking:

Reply:
Ok, lets do this in master